flow/scripts/synth.tcl

```diff
@@ -67,8 +67,16 @@ if {[info exist ::env(BLOCKS)]} {
 synth  -top $::env(DESIGN_NAME) {*}$::env(SYNTH_ARGS)
 
 if { [info exists ::env(USE_LSORACLE)] } {
+    set lso_script [open $::env(OBJECTS_DIR)/lso.script w]
+    puts $lso_script "ps -a"
+    puts $lso_script "oracle --config $::env(LSORACLE_KAHYPAR_CONFIG)"
+    puts $lso_script "ps -m"
+    puts $lso_script "crit_path_stats"
+    puts $lso_script "ntk_stats"
+    close $lso_script
+
     # LSOracle synthesis
-    lsoracle
+    lsoracle -script $::env(OBJECTS_DIR)/lso.script -lso_exe $::env(LSORACLE_CMD)
```

Comment:
This might require an update to LSoracle, but I get a file "hypergraph.txt" in my directory. The executable shouldn't be generating random files, if this file is not needed, then it should not get printed. If it's needed, it should probably go to the tmp directory and get cleaned up on exit.

Reply:
I'll need to investigate why that is being generated. I've filed an issue https://github.com/lnis-uofu/LSOracle/issues/51